sdk/core/perf-tests/core-rest-pipeline/README.md

```diff
@@ -1,9 +1,17 @@
 ### Guide
 
 1. Build the core-rest-pipeline perf tests package `rush build -t perf-core-rest-pipeline`.
-3. Copy the `sample.env` file and name it as `.env`.
-4. Populate the `.env` file with your Azure Credentials.
-5. Refer to the [rate limits](https://docs.microsoft.com/azure/active-directory/enterprise-users/directory-service-limits-restrictions) and then run the tests as follows:
+2. Copy the `sample.env` file and name it as `.env`.
+3. Populate the `.env` file with your Azure Credentials.
+4. Refer to the [rate limits](https://docs.microsoft.com/azure/active-directory/enterprise-users/directory-service-limits-restrictions) and then run the tests as follows:
+5. `rushx build` to compile the TS tests to JS.
```

Comment:
I thought the perf-test package.json script took care of this already.

Reply:
Good point